File: Python/Examples/update_env.py
import requests

url = "https://api.getpostman.com/environments/{{environment_id}}"

payload = "{\n    \"environment\": {\n        \"values\": {{updated_environment}}\n    }\n}"
headers = {
    'x-api-key': "{{postman_api_key}}",
    'content-type': "application/json"
    }

response = requests.request("PUT", url, data=payload, headers=headers)

print(response.text)

File: Ruby/Examples/UpdateEnv.rb
require 'uri'
require 'net/http'

url = URI("https://api.getpostman.com/environments/{{environment_id}}")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Put.new(url)
request["x-api-key"] = '{{postman_api_key}}'
request["content-type"] = 'application/json'
request.body = "{\n    \"environment\": {\n        \"values\": {{updated_environment}}\n    }\n}"

response = http.request(request)
puts response.read_body